### ISSUE TITLE
Can't start DEV

### ISSUE THREAD

**Issue by SonGokussj4** [siefkenj/react-userscripts#9]
I'm on Windows 10, running from Terminal - Powershell

```
C:\GIT\react-userscripts\userscript on  master[?!] via node
➜  npm start

> react-userscripts@1.1.0 start
> PORT=8124 BROWSER=none FAST_REFRESH=false WDS_SOCKET_HOST=localhost react-app-rewired start

'PORT' is not recognized as an internal or external command,
operable program or batch file.
```

**Comment by siefkenj** [comment]
I see. This script assumed it was running in `BASH`. I don't have any Windows system to test anything with...

Can you run it inside of Windows Subsystem for Linux? It should work there.

**Comment by bosam** [comment]
One solution is to include [cross-env](https://www.npmjs.com/package/cross-env).
```npm install --dev cross-env```
and replace
```"start": "PORT=8124 BROWSER=none FAST_REFRESH=false WDS_SOCKET_HOST=localhost react-app-rewired start",```
with
```"start": "cross-env PORT=8124 BROWSER=none FAST_REFRESH=false WDS_SOCKET_HOST=localhost react-app-rewired start",```

This would handle windows and any other environments to support environment variables as start arguments.

**Comment by siefkenj** [comment]
Thanks @bosam   This issue should be resolved as of PR #10
